drivers/wifi/nrf700x/zephyr/src/qspi/src/qspi_if.c

```diff
@@ -289,10 +322,25 @@ static inline void qspi_lock(const struct device *dev)
 #else /* CONFIG_MULTITHREADING */
 	ARG_UNUSED(dev);
 #endif /* CONFIG_MULTITHREADING */
+
+	/*
+	 * Change the base clock divider only for the time the driver is locked
+	 * to perform a QSPI operation, otherwise the power consumption would be
+	 * increased also when the QSPI peripheral is idle.
+	 */
```

Comment:
indent looks odd

Reply:
These lines are copied from the Zephyr 'nrf_qspi_nor.c'...

Is the following lines better?

	/* Change the base clock divider only for the time the driver is locked
	 * to perform a QSPI operation, otherwise the power consumption would be
	 * increased also when the QSPI peripheral is idle.
	 */

Any suggestion?

Reply:
Yes, this looks fine.